ui/apps/dashboard/src/app/baseWrapper.tsx

```diff
@@ -9,7 +9,10 @@ export function BaseWrapper({ children }: { children: React.ReactNode }) {
       lang="en"
       className={cn('h-full bg-white accent-indigo-500', interTight.variable, robotoMono.variable)}
     >
-      <body className="h-full overflow-hidden">{children}</body>
+      <body className="h-full overflow-hidden">
```

Comment:
At some point we should think if it's worth replacing this with the shared `<AppRoot>` component, but I don't wanna mess with layouts now